### PR TITLE
[839] Port the terms interruption page

### PR DESCRIPTION
**Pull request by darokel**
### Context

- https://trello.com/c/f3sjnl8o/839-migrate-the-interruption-pages-accept-terms

### Changes proposed in this pull request

- Rework the terms and conditions logic
- Add test coverage

### Guidance to review

View the review app and login with the Colin persona. You should be taken to the accept terms page.

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: app/forms/publish/accept_terms_form.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module Publish
+  class AcceptTermsForm < BaseModelForm
```
Comment (defong) [inline app/forms/publish/accept_terms_form.rb:2]: I think we should namespace this a bit more

```suggestion
module Interruption
  class AcceptTermsForm < BaseModelForm
```

File: app/controllers/terms_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class TermsController < ApplicationController
```
Comment (defong) [inline app/controllers/terms_controller.rb:1]: missing? module Publish
```suggestion
module Publish
class TermsController < ApplicationController
```

**Review by defong — APPROVED** [review]

File: app/forms/publish/accept_terms_form.rb (hunk 1 above)
Comment (darokel) [reply app/forms/publish/accept_terms_form.rb]: done

File: app/controllers/terms_controller.rb (hunk 2 above)
Comment (darokel) [reply app/controllers/terms_controller.rb]: done

**Pull request merged**